src/Chunk.cpp

```diff
@@ -621,31 +622,37 @@ void cChunk::Tick(std::chrono::milliseconds a_Dt)
 
 	for (cEntityList::iterator itr = m_Entities.begin(); itr != m_Entities.end();)
 	{
```

Comment:
@SphinxC0re I essentially mirror your exact thought process every time I see this.

@LogicParrot Perhaps you could auto-ify the type if you do a rebase later? There's no need to inconvenience yourself by doing this individually.